### ISSUE TITLE
Internal Error: SQL Error: SQLSTATE[HY000]: General error: 1 no such table: main.task_has_subtasks

### ISSUE THREAD

**Issue by wzhik** [kanboard/kanboard#4858]
### Actual behaviour
When trying to complete a subtask, an error appears about a missing table in the database 
Internal Error: SQL Error: SQLSTATE[HY000]: General error: 1 no such table: main.task_has_subtasks

### Steps to reproduce
1. Create a task
2. Create a subtask
3. Click the "done" checkbox and get this error 

### Screenshots
https://i.imgur.com/bS77NJb.png

### Configuration

https://i.imgur.com/vfLe3M0.png

- Kanboard version:  1.2.20
- Database type and version: sqllite
- PHP version: 7.4
- OS: Linux Mint
- Browser: Firefox 90

**Comment by fguillot** [comment]
See https://kanboard.discourse.group/t/error-no-such-table-main-task-has-subtasks/1664

Duplicate with #4736 and #1698

**Comment by evolu1** [comment]
> https://kanboard.discourse.group/t/error-no-such-table-main-task-has-subtasks/1664/15

Workaround solution from there:

```
ALTER TABLE subtask_time_tracking RENAME TO subtask_time_tracking_old;

CREATE TABLE subtask_time_tracking (
    "id"	INTEGER,
    "user_id"	INTEGER NOT NULL,
    "subtask_id"	INTEGER NOT NULL,
    "start"	INTEGER DEFAULT 0,
    "end"	INTEGER DEFAULT 0,
    "time_spent"	REAL DEFAULT 0,
    FOREIGN KEY("user_id") REFERENCES "users"("id") ON DELETE CASCADE,
    FOREIGN KEY("subtask_id") REFERENCES "subtasks"("id") ON DELETE CASCADE
);

INSERT INTO subtask_time_tracking SELECT * FROM subtask_time_tracking_old;
```

**These SQL statement worked and fixed the issue for me!**
It was new installation of kanboard-1.2.20.

To run these SQL statements in windows in the sqlite database:

1. copy the SQL statements into a textfile and name it `fix.sql`
2. download the `db.slite` database from `/kanboard/data`
3. download the `sqlite-tools-win32-x86-3360000.zip` (1.82 MiB) from https://sqlite.org/download.html and unzip
4. place the `db.slite` and `fix.sql` next to `sqlite3.exe`
5. open cmd and run `sqlite3.exe db.slite` and then run `.read fix.sql`
6. if no error shows, then it should have worked, close the window
7. place the `db.slite` back to `/kanboard/data`

**Comment by Keex0r** [comment]
If you're ok with third party software, there is also [DB Browser for SQLite](https://github.com/sqlitebrowser/sqlitebrowser), which is quite handy for playing around with the databases :-)

**Comment by bttrx** [comment]
The fix by @Keex0r worked for me in version 1.2.21, but I wonder, why the problem is still not fixed om the release code.

**Comment by jonatasartur** [comment]
This issue is occurring in release 1.2.22 and through gitclone.
This issue is fixed by #4978
